common/network-common/src/main/java/org/apache/spark/network/crypto/TransportCipher.java

```diff
@@ -84,7 +85,8 @@ SecretKeySpec getKey() {
     return outIv;
   }
 
-  private CryptoOutputStream createOutputStream(WritableByteChannel ch) throws IOException {
+  @VisibleForTesting
```

Comment:
You use a `VisibleForTesting` annotation to release modifier `private`, just for junit?
Maybe Encapsulation a top class used for junit is better?

Reply:
This is package private. We usually use this to expose methods for junit. It's not worth to encapsulate a class just for unit tests.

Reply:
OK, I get it.